lib/vanagon/utilities.rb

```diff
@@ -147,6 +147,7 @@ def retry_with_timeout(tries = 5, timeout = 1, &blk)
 
       message = "Block failed maximum number of #{tries} tries"
       message += "\n with error #{error.message}" unless error.nil?
```

Comment:
Nitpick. Code Repetition. Refactor with

```ruby
unless error.nil?
  message +=  "\n with error #{error.message}" +
                        "\n#{error.backtrace.join("\n")}
end
```

Reply:
maybe this will make Rubocop happier too 😊